ze/tracer_ze_helpers.include.c

```diff
@@ -990,14 +993,14 @@ void readCopyE(uint32_t deviceIdx, copyEngineData *copyData ){
     result = ZES_ENGINE_GET_PROPERTIES_PTR(_sampling_engineHandles[deviceIdx][j], &engineProp);
```

Comment:
You may want to tore these, rather than query them everytime, as they will not change.